### DISCUSSION TITLE
Publishing and/or Blogging?

### DISCUSSION THREAD

**Discussion by berot3** [GitJournal/GitJournal#574] [General]
Hi!

This looks interesting! I was wondering if anyone is using GitJournal to also publish the notes to a Hugo site or similar? 

Also: is GitJournal focusing only on note-taking or do you also have features to help writing journal/blog? For example a button to write a post for a day with tag-support.

**Comment by vHanda** [comment]
Hello. I occasionally use GitJournal for minor edits to https://gitjournal.io - Hugo based.

I didn't build GitJournal with blogging in mind, but since it has been a popular use-case I've slowly added features to integrate with different static website generators. I'm not too sure what you mean by "A post for a day with tag-support", if you could please elaborate, I'll be happy to implement it.

On a separate note, I started working on another app focused exclusively for Hugo. It's mostly adapted from the GitJournal code base. The main difference is that it embeds "hugo" inside the app, so the "preview" is the real "preview" of the website. I'm just struggling with doing an initial "npm install", as packaging npm + node for Android / ios is proving to be difficult. Maybe that part can be done on the server side.

Also, I'm not too sure if it makes sense for that app to be Open Source and the respective business model. I'm leaning towards keeping it proprietary and having an x-days trial, with a purchase option (one time, no subscription)
